### DISCUSSION TITLE
ACE Medical - Not Working Appropriately in Multiplayer

### DISCUSSION THREAD

**Discussion by mattdimegs** [acemod/ACE3#9852] [Q&A]
Hey all, 

My milsim unit has been having problems with not being able to treat others. During the operation, we have problems where if you treat someone else while they are treating you, whoever finishes their task first is completed and the other persons just drops.

For example, if I was treating patient A for a 6 second treatment, and he tries the same treatment on himself on a different part of the body, his action would stop when my completes.

Is this intended?
Thanks!

**Comment by johnb432** [comment]
I had this in the past when I was testing a mod, it was either "Fire Support Plus" (version that had "Chemical Warefare" integrated).

https://github.com/assaultboy/ChemicalWarfare/blob/master/addons/CBRN_gear/functions/fn_treatmentHandler.sqf is what caused the issue. I don't know if "Fire Support Plus" has changed their code since my last test (my test was over 2 years old).

If you can reproduce the problem with CBA + ACE only, open an issue. Discussions is not the place for this.
